user/docker.md

```diff
@@ -175,6 +178,19 @@ before_install:
   - sudo mv docker-compose /usr/local/bin
 ```
 
+### Installing a newer Docker version
+
+The Trusty sudo-required environment comes with Docker 1.12 pre-installed.
```

Comment:
I think I wouldn't include a version number as it will probably change in the next environment update. Maybe something along the line of the following would be better?

> The pre-installed Docker version is possibly not the latest. You can upgrade to ...